tests/response-generator/index.js

```diff
@@ -396,6 +416,105 @@ describe('Response Generator', () => {
 			});
 		});
 
-	});
+		it('Should return a generated response with value generated using relevant faker method if x-faker extension is ' +
+			'present in and method exists in faker', () => {
+			sinon.replace(faker.name, 'firstName', sinon.fake.returns('bob'));
```

Comment:
I'd suggest to use `sinon.stub()` here and in next test case for consistency between tests.
```suggestion
			sinon.stub(faker.name, 'firstName').returns('bob');
```

Reply:
Yeah that's a good idea.
I think I started with `replace` and `fake` since I think that is mostly what the `sinon` project is moving towards but still lacks certain functionality using those methods.
I agree, switching to be consistent results in cleaner code.